cluster/saltbase/salt/kubelet/default

```diff
@@ -41,6 +41,12 @@
 {% endif -%}
 
 {% set config = "--config=/etc/kubernetes/manifests" -%}
+
+{% set manifest_url = "" -%}
+{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
```

Comment:
Is there a reason to only do this on the master? We will (hopefully soon) treat the master node on GCE just like any other node, so we should look at reducing the number of special cases we apply to the master.